packages/ia-topnav/test/media-slider.test.js

```diff
@@ -6,7 +6,8 @@ import { texts } from '../src/data/menus';
 const component = (mediaSliderOpen, selectedMenuOption) => (
   html`<media-slider
           ?mediaSliderOpen="${mediaSliderOpen}"
-          selectedMenuOption="${selectedMenuOption}"></media-slider>`
+          selectedMenuOption="${selectedMenuOption}"
```

Comment:
QUESTION: Is there a reason this parameter is treated as an attribute and not a property, e.g. `.selectedMenuOption`? Seems like in most other places in this code, it's treated as a JS property.

Reply:
No, there's no reason. It was simply omitted. It will behave the same, however, since the default attribute converter is being used.